src/node_os.cc

```diff
@@ -223,6 +230,12 @@ static void GetLoadAvg(const FunctionCallbackInfo<Value>& args) {
   Local<ArrayBuffer> ab = array->Buffer();
   double* loadavg = static_cast<double*>(ab->GetContents().Data());
   uv_loadavg(loadavg);
+
+#if ENABLE_TTD_NODE
+  if(s_doTTRecord || s_doTTReplay) {
```

Comment:
here too

src\node_os.cc:235:  Missing space before ( in if(  [whitespace/parens] [5]